lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnSubmissionHelper.java

```diff
@@ -136,12 +139,11 @@ public YarnSubmissionHelper setApplicationName(final String applicationName) {
   }
 
   /**
-   * Set the amount of memory to be allocated to the Driver.
-   * @param megabytes
+   * Set the resources for the Driver.
```

Comment:
Set memory and number of cores to be allocated to the driver.

Reply:
Thanks, I addressed this.